starter-project-backend-g31/src/server.ts

```diff
@@ -2,7 +2,7 @@ import app from "./app"
 import mongoose from 'mongoose';
 
 const PORT = process.env.PORT || 8000
-const DB_URI = process.env.MONGO_URI || "mongodb://localhost:27017/test";
+const DB_URI = process.env.mongodb_url || "mongodb://localhost:27017/rating";
```

Comment:
make the variable naming better and readable.. like MONGO_URI and MONGO_URI_TEST bla bla :)